workflow/rocoto/gfs_tasks.py

```diff
@@ -927,9 +927,12 @@ def atmanlprod(self):
         return task
 
     @staticmethod
-    def _get_ufs_postproc_grps(cdump, config):
+    def _get_ufs_postproc_grps(cdump, config, component='atmos'):
 
         fhrs = Tasks._get_forecast_hours(cdump, config)
```

Comment:
`Tasks._get_forecast_hours()` is providing the atmos list of forecast hours (using the HF period), but in the forecast job the HF setting is not used for the marine output times.

Reply:
fixed.